js/console.ts

```diff
@@ -122,6 +122,8 @@ function stringify(
     case "string":
       return value;
     case "number":
+      // Special handling of -0
+      return Object.is(value, -0) ? "-0" : `${value}`;
```

Comment:
Does node do this?

Reply:
Yes, https://github.com/nodejs/node/blob/8876ac5c358114f0f88424f6737ca4b89fc9e6c7/lib/internal/util/inspect.js#L1026-L1029